cmd/bundle/generate/job.go

```diff
@@ -83,7 +85,14 @@ func NewGenerateJobCommand() *cobra.Command {
 			return err
 		}
 
-		filename := filepath.Join(configDir, fmt.Sprintf("%s.yml", jobKey))
+		oldFilename := filepath.Join(configDir, fmt.Sprintf("%s.yml", jobKey))
```

Comment:
Can you add a comment on why we are renaming the file here? I presume it's to support the iterative user case where users update a job in the UI and then generate to get the latest changes. We should also self link this PR and call out that the renaming code was added to make this change backwards compatible.